.github/scripts/get-job-id.ts

```diff
@@ -0,0 +1,41 @@
+import { retry } from './shared/utils';
+import * as core from '@actions/core';
+
+async function main() {
+  const { Octokit } = await import('octokit');
+
+  const env = {
+    OWNER: process.env.OWNER || 'metamask',
+    REPOSITORY: process.env.REPOSITORY || 'metamask-extension',
+    RUN_ID: +process.env.RUN_ID!,
+    JOB_NAME: process.env.JOB_NAME!,
+    ATTEMPT_NUMBER: +process.env.ATTEMPT_NUMBER!,
+    GITHUB_TOKEN: process.env.GITHUB_TOKEN!,
+    GITHUB_ACTIONS: process.env.GITHUB_ACTIONS === 'true',
+  };
```

Comment:
non blocking nit, just FYI: I don't think this section is being linted as I don't think we allow the `Non-null assertion operator`.